### ISSUE TITLE
Rewrite in electron

### ISSUE THREAD

**Issue by thetooth** [siavash119/qtchan#7]
Hello my friend I hope this issue finds you well.

As a regular frog poster I feel as though a clunky, legacy interface such as Qt will be the primary bottleneck for shit posting. I propose we move to a more modern and scalable interface such as Electron.

I can assist by developing the responsive front end code in HTML5 and CSS3 (material design :+1:). My friend highly recommends React as a framework(he works at Google :1st_place_medal:) as it meets the web scale demand I'm sure we'll see :ok_hand::ok_hand::ok_hand:.

I also have some ideas on user contribution, why force everyone to use PR's to provide new features? I propose we implement a client side module loading routine that dynamically builds and loads NPM packages starting with 4chan*, this way anyone can design and publish new features! :dark_sunglasses:

**Comment by siavash119** [comment]
You're absolutely correct: our only hope for accelerated shit posting will be through future GUI libraries and frameworks. Yes, Electron and React would be excellent choices. As we know, Electron brings all the features of Chrome, and React copies the entire DOM structure into memory, allowing us to perform all the loading and manipulations we need in an ever scaling environment. And to develop faster I agree as well: I'll host the source on an ftp server with guest write access, so anybody can help.

**Comment by gergesh** [comment]
If I may chime in on this - I'd hate it if this project turned to electron. If I wanted a browser experience I'd just use my browser, there are plenty out there.

**Comment by Mashironi** [comment]
I know this was supposed to be a joke, but if you ever consider using Electron in the future, please don't. ;_;

**Comment by xdevelnet** [comment]
You guyz are so funny. Keep on, please.

**Comment by wannaseeanalien** [comment]
That's a no from me.
OP is not smart.
